### PR TITLE
Release 11.3

### PR DESCRIPTION
**Pull request by swethakundaram**

### CODE REVIEW

File: learn/teams/code-repository.md [hunk 1]
```diff
@@ -34,33 +34,54 @@ For more information, see [Push to External Repo](/learn/app-development/dev-int
...
+![Edit Repository](/learn/assets/wm_coderep_6_f.png)
 
-## Make Code Repository as Default
+## Set Branch as Default
 
-* Select the Code Repository you want to make as default, and click the **Default** option.
+In this field you add the branch name that is to be added as default branch while creating repository. With this, you can avoid setting the master branch as default and add desired branch as default.
 
-![Make Repository Default](/learn/assets/TeamMakeVcsDefault.png)
+![Set Repository Default](/learn/assets/wm_coderep_5_f.png)
```
Comment (swethakundaram) [inline learn/teams/code-repository.md:94]: If you are replacing the image, please the same image name and remove the old image.

File: learn/app-development/widgets/prefab/enterprise-marketplace.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+Enterprise Marketplace lets teams published artifacts for reuse across the team. Developer create artifacts and team admin approves and publishes the version of artifact to Marketplace via Teamportal. 
+
+![Enterprise Marketplace](/learn/assets/wm_enterprisemarketplace.png)
+
+You can view the list of prefabs to be published to Enterprise Marketplace in Manage Prefabs page.
+
+![Enterprise Marketplace](/learn/assets/wm_marketplaceprefabs.png)
+
+:::note
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/prefab/enterprise-marketplace.md:17]: We should define the roles involvled in this activity clearly before using them in sentence.

File: learn/app-development/widgets/prefab/enterprise-marketplace.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+Any team that has been given access by the super admin can publish the artifacts to Enterprise Marketplace where super admin has complete access over platform.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/prefab/enterprise-marketplace.md:18]: We should not use the term super admin.

File: learn/app-development/widgets/prefab/enterprise-marketplace.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+:::
+
+## Publisher Information
+
+Publishing artifacts allow multiple teams access the artifacts published in **Marketplace**. Publisher information includes the publishing team name. Every artifact is associated with publisher information.
+
+![Publisher Information](/learn/assets/wm_artifactpublisherdetails.png)
+
+:::note
+You can only publish an artifact using semantic versioning to mention the version of the artifact.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/prefab/enterprise-marketplace.md:32]: Rewrite

File: learn/app-development/widgets/prefab/enterprise-marketplace.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+:::
+
+## Semantic Versioning
+
+Semantic versioning is the standard three digit versioning system to publish an artifact that includes, Major.Minor.Patch details of respective artifact. Every project branch can have its own sequence of versioning in ascending order. Versions within the project are unique across branches and are validated by WaveMaker.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/prefab/enterprise-marketplace.md:37]: Rewrite

File: learn/on-premise/configure/config-users-auth-providers.md [hunk 6]
```diff
@@ -74,3 +74,16 @@ The User should be mapped with a respective team to access the studio. Following
 
 
 [![authentication provider](/learn/assets/wme-setup/configuring-wme/adding-authorization-provider.png)](/learn/assets/wme-setup/configuring-wme/adding-authorization-provider.png)
+
+### Restricting Login for all users
+
+You can restrict the users from logging into WaveMaker studio by enabling Restrict Login for all users option. Once enabled, all active users get logged out of the platform within three minutes. Only the super admin who enables this restriction stays logged in.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/on-premise/configure/config-users-auth-providers.md:80]: WE should explain why this was introduced

Comment (prasanth-reddy-wavemaker-com) [inline learn/on-premise/configure/config-users-auth-providers.md:80]: Super admin should not be used.

File: learn/teams/code-repository.md [hunk 7]
```diff
@@ -34,33 +34,61 @@ For more information, see [Push to External Repo](/learn/app-development/dev-int
...
 
+## Add Access Type
+
+This field restricts the VCS access to the members of the team. With this, admin allows the members to use necessary VCS operations. Two types of access options are provided for a code repository.
+
+- **Service Account**: In this type, all project members access the repository with same credentials as the team admin.
+- **Developer Credentials**: In this type, every member of the project use individual credentials to access the project and its VCS operations are allowed based on member respective permissions to VCS.
+
+:::important
+When Developer Credentials is selected as Access type for the Code Repository, authentication credentials are required to perform any VCS operation. If you do not have the credentials or expired the credentials, then below screen is displayed to update your credentials.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/teams/code-repository.md:67]: Remove authentication just say credentials

File: learn/teams/code-repository.md [hunk 8]
```diff
@@ -34,33 +34,61 @@ For more information, see [Push to External Repo](/learn/app-development/dev-int
...
+
+![Add Access Type](/learn/assets/wm_coderep_7_f.png)
+
+:::
+
+![Add Access Type](/learn/assets/wm_coderep_3_f.png)
+
+## Delete Repository from VCS
+
+This field is to enable deletion of a repository from VCS when deleting the project from WaveMaker Studio. Third party users can disable this option and store the project details over VCS without any dependency on studio.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/teams/code-repository.md:77]: Just say "Enables deletion...' no need to start with This field is to enable.

File: learn/teams/code-repository.md [hunk 9]
```diff
@@ -34,33 +34,61 @@ For more information, see [Push to External Repo](/learn/app-development/dev-int
...
+- **Delete**: To delete the VCS details from Code Repository page.
 
-![Add Repository](/learn/assets/TeamAddVcsRep.png)
+![Edit Repository](/learn/assets/wm_coderep_6_f.png)
 
-## Make Code Repository as Default
+## Set Branch as Default
 
-* Select the Code Repository you want to make as default, and click the **Default** option.
+In this field you add the branch name that is to be added as default branch while creating repository. With this, you can avoid setting the master branch as default and add desired branch as default.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/teams/code-repository.md:92]: Rewrite.

Want to choose a branch other than master as the branch WaveMaker will push code to? Then just mention that branch name here.

File: learn/teams/manage-projects.md [hunk 10]
```diff
@@ -18,29 +18,47 @@ Members should be registered with **Teams** before they can be invited to a proj
...
-- View or edit a user’s role in a project  
-- View project information  
-- Remove a user from a project
+- View or edit user’s role in a project  
+- Add and view project information  
+- Remove a user from project
+
+## Import Project from Code Repository
+
+You can import a project available on Version Control Systems (VCS) like GitHub, GitLab, Bitbucket and Azure. It also imports commit history and specified branches of the project.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/teams/manage-projects.md:34]: Rewrite

It also imports should be replaced with. Commit history of the specified branch carries over when imported.

**Review by prasanth-reddy-wavemaker-com — CHANGES_REQUESTED** [review]

**Review by swethakundaram-wm — APPROVED** [review]

**Review by prasanth-reddy-wavemaker-com — APPROVED** [review]

**Pull request merged**